env/detect_machine.sh

```diff
@@ -96,6 +96,8 @@ case $(hostname -f) in
 
   login01.expanse.sdsc.edu) MACHINE_ID=expanse ;; ### expanse1
   login02.expanse.sdsc.edu) MACHINE_ID=expanse ;; ### expanse2
+
+  nid0*) MACHINE_ID=odin ;; ### expanse2
 esac
```

Comment:
@ywangwof, could you change the comment here to reflect the login node name (or names) for Odin?  Thanks!

Reply:
Also, based on the structure for other machines, I think it's best to be explicit and list all nid0 values, instead of the wildcard.  Thanks!